task3/tableTimer.js

```diff
@@ -0,0 +1,78 @@
+const DOWNLOADS = [
+  {
+    id: 1,
+    title: "Recipe",
+    status: "Done",
+  },
+  {
+    id: 2,
+    title: "Workouts",
+    status: "Pending",
+  },
+  {
+    id: 3,
+    title: "Passwords",
+    status: "Pending",
+  },
+  {
+    id: 4,
+    title: "To Do 2021",
+    status: "Pending",
+  },
+  {
+    id: 5,
+    title: "Books",
+    status: "Failed",
+  },
+];
+function constructTable(array) {
+  let table = document.createElement("table");
+  document.getElementsByTagName("body")[0].appendChild(table);
+
+  for (let i = 0; i < array.length; i++) {
+    let element = array[i];
+
+    let row = document.createElement("tr");
+    let cell = document.createElement("td");
+    let cellText = document.createTextNode(element.id);
+    cell.appendChild(cellText);
+    row.appendChild(cell);
+    let cell = document.createElement("td");
+    let cellText = document.createTextNode(element.title);
+    cell.appendChild(cellText);
+    row.appendChild(cell);
+    let cell = document.createElement("td");
+    let cellText = document.createTextNode(element.status);
+    cell.appendChild(cellText);
+    row.appendChild(cell);
+
+    table.appendChild(row);
+  }
+}
+
+constructTable(DOWNLOADS);
+
+const START_DELAY = 3000;
+const STATUS_CHECK_INTERVAL = 5000;
+const INITIAL_STATUS = "Pending";
+const FINAL_STATUS = "Done";
```

Comment:
It will be better to organize statuses in one collection:
```
const DOWNLOAD_STATUSES = {
  pending: 'Pending',
  done: 'Done',
  failed: 'Failed',
}
```